### ISSUE TITLE
Simplify the using AcrylicBrush 

### ISSUE THREAD

**Issue by wuyang26** [sourcechord/FluentWPF#25]
Just I install the "Windows Community Toolkit Sample App",in the sample ,using the liked AcrylicBrush  is easy:
     <Border BorderBrush="Black" BorderThickness="1"
              Grid.Column="2"
              Height="400">
        <Border.Background>
               <media:BackdropBlurBrush Amount="7.95" />
        </Border.Background>
      </Border>
so can you cancel the target assign?
the other needs,can you design the full theme like the "Xaml controls gallery"? it is very cool!

**Comment by wuyang26** [comment]
the Windows Community Toolkit is open source,you can find the realize code in github for BackdropBlurBrush .

**Comment by sourcechord** [comment]
Hi, @wuyang26.
BackdropBlurBrush is based on UWP specific feature(XamlCompositionBrushBase).
Maybe, it is impossible to simulate these feature in WPF platform.
This is why AcrylicBrush needs `target` property.

By the way, I'll implement more theme style for some controls(CheckBox, RadioButton, ListView...etc).
Please, wait for a while.

**Comment by wuyang26** [comment]
Ok,I konw the BackdropBlurBrush is based on UWP,I just holp the AcrylicBrush  is perfect.
By the way,the popup control can use the system  Acrylic API by the attach behaviour!

**Comment by sourcechord** [comment]
Sounds great :+1:
Acrylic popup will be very cool.

Just now, I'm trying to implement acrylic context menu.
I think there is more usecase for context menu rather than Popup.

In the first place, implement AcrylicContextMenu.
Then, I'll implement acrylic Popup class.

**Comment by wuyang26** [comment]
OK,I holp you realize the contextmenu,but it is difficult.
I try the menu once,but faild,the menu’s space is full screen,but visible area.

发送自 Windows 10 版邮件<https://go.microsoft.com/fwlink/?LinkId=550986>应用

________________________________
发件人: minami_SC <notifications@github.com>
发送时间: Thursday, October 18, 2018 9:51:12 PM
收件人: sourcechord/FluentWPF
抄送: wuyang26; Mention
主题: Re: [sourcechord/FluentWPF] Simplify the using AcrylicBrush (#25)


Sounds great 👍
Acrylic popup will be very cool.

Just now, I'm trying to implement acrylic context menu.
I think there is more usecase for context menu rather than Popup.

In the first place, implement AcrylicContextMenu.
Then, I'll implement acrylic Popup class.

**Comment by sourcechord** [comment]
Thanks for your information.
This is current progress of my work.

![image](https://user-images.githubusercontent.com/14825436/47536513-6bb1b780-d8fb-11e8-9da4-a651ad7502e1.png)

It was certainly difficult, but I have good prospect for achieve it.
AcrylicContextMenu will included future updates(maybe until the end of next month).

**Comment by wuyang26** [comment]
Congratulations，there are two problems:

1>    Fluent API  can’t apply animation,for example:popup animation

2>    Can you decompile the uwp Dll? We don’t how to realize the AcrylicBrush by the microsoft for uwp ,I think they use the fluent api too.

But you don’t, I worry about the performance.
发送自 Windows 10 版邮件<https://go.microsoft.com/fwlink/?LinkId=550986>应用

________________________________
发件人: minami_SC <notifications@github.com>
发送时间: Friday, October 26, 2018 7:46:36 AM
收件人: sourcechord/FluentWPF
抄送: wuyang26; Mention
主题: Re: [sourcechord/FluentWPF] Simplify the using AcrylicBrush (#25)


Thanks for your information.
This is current progress of my work.

[image]<https://user-images.githubusercontent.com/14825436/47536513-6bb1b780-d8fb-11e8-9da4-a651ad7502e1.png>

It was certainly difficult, but I have good prospect for achieve it.
AcrylicContextMenu will included future updates(maybe until the end of next month).

―

**Comment by sourcechord** [comment]
Hi, @wuyang26 
Thank you for the information!!

1> 
I forgot PopupAnimation property of Popup control.
So, I tried to apply Fluent API for Popup.
But, PopupAnimation doesn't work properly.

2>
Sorry, I don't have skills for decompile and so on.
(Perhaps reverse engineering is prohibited by EULA...)

However, AcrylicMenu&AcrylicPopup's performance is not so bad, except there is no Popup's animation.
I'll implement AcrylicPopup with Fluent API, that is omitted PopupAnimation behavior.

**Comment by wuyang26** [comment]
You can set the property “PopupAnimation" for “Slide”,when run the demo,the Acrylic effect appear immediately,
but the content slide appear, so it is not synchronized.
However,the windows10’infomation popup is OK.
The follow code from the telerik(they don’t resolve the Acrylic  Animation,too):
<Popup mat:ThemeEffectsHelper.IsAcrylic="True" StaysOpen="True" AllowsTransparency="True" PopupAnimation="Slide">
发件人: minami_SC<mailto:notifications@github.com>
发送时间: 2018年11月14日 20:40
收件人: sourcechord/FluentWPF<mailto:FluentWPF@noreply.github.com>
抄送: wuyang26<mailto:wuyang26@live.cn>; Mention<mailto:mention@noreply.github.com>
主题: Re: [sourcechord/FluentWPF] Simplify the using AcrylicBrush (#25)


Hi, @wuyang26<https://github.com/wuyang26>
Thank you for the information!!

1>
I forgot PopupAnimation property of Popup control.
So, I tried to apply Fluent API for Popup.
But, PopupAnimation doesn't work properly.

2>
Sorry, I don't have skills for decompile and so on.
(Perhaps reverse engineering is prohibited by EULA...)

However, AcrylicMenu&AcrylicPopup's performance is not so bad, except there is no Popup's animation.
I'll implement AcrylicPopup with Fluent API, that is omitted PopupAnimation behavior.

―

**Comment by sourcechord** [comment]
Hi, @wuyang26.

Sorry for very late responding..
I created AcrylicMenu&AcrylicContextMenu in FluentWPF 0.6.1
And, of cource I tried with `PopupAnimation="Slide"`.

Slide animation is provided on the WPF's visual layer.
On the other hand, Acrylic effect is drawn on the Native windows layer.
Acrylic effect couldn't work with PopupAnimation.
It is impossible to simulate PopupAnimation on the WPF's framework.

There is many difference to create window during UWP & WPF.
UWP & WPF are created in different technology.

So that, I'd like to decide that is limitations for FluentWPF library.
